src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
Yeah, I saw that discussion in your other PR (I think Chris posted it in a channel) and it makes sense.  So we're saying a given action may result in different reducers updating different types of state, but the action doesn't/shouldn't need to know about that.

Sounds like a fun and messy refactoring project that will hold our testing approach to the fire!